plugin/trino-faker/src/test/java/io/trino/plugin/faker/FakerQueryRunner.java

```diff
@@ -47,6 +50,12 @@ protected Builder()
                     .build());
         }
 
+        public Builder setProperties(Map<String, String> properties)
```

Comment:
Rename this method to `setConnectorProperties`. The builder already has `setExtraProperties` and  `addCoordinatorProperty`. The current method name looks confusing.

Reply:
setProperties -> setFakerProperties

Reply:
Thank you, these are great suggestions!